harvard-huit/Harvard-Geospatial-Library
# Modal window restyling

**Modal window restyling**
* * *

**JIRA Ticket**: [HGL-468: update styles of modal windows](https://jira.huit.harvard.edu/browse/HGL-468)

# What does this Pull Request do?
Modal popup windows were restyled to reflect our branding and be more consistent with one another.

# How should this be tested?
Compare modal windows with their marvel mockups and/or dev site counterparts and look out for the following:
* Citation modal ([mockup](https://marvelapp.com/prototype/c0j56e8/screen/79048570) and examples on [local](https://localhost:3001/catalog/stanford-fb897vt9938) and [hgl-dev](https://hgl-dev.lib.harvard.edu/catalog/stanford-fb897vt9938)) - to view modal, click on "Cite" link in sidebar
   * Smaller heading
   * Citation text in lora font
   * URLs are hyperlinked and same font + font-size as rest of text (Lora, 15px)
   * After bookmarking a few items, the cite link on the bookmark page will have spacing between each item instead of being a giant blob of text
* HGL request modal (examples on [local](https://localhost:3001/catalog/harvard-ams7810-s250-u54-ng50-13) and [hgl-dev](https://hgl-dev.lib.harvard.edu/catalog/harvard-ams7810-s250-u54-ng50-13))- to view modal, click on "Download Geotiff" button
   * vertically centered form fields
   * restyled text, label, and input box
* Metadata modal ([mods tab mockup](https://marvelapp.com/prototype/c0j56e8/screen/79049934) and [iso tab mockup](https://marvelapp.com/prototype/c0j56e8/screen/79050342)) - to view modal, click on "Metadata" link in sidebar
   * [MODS and ISO tabs](https://localhost:3001/catalog/stanford-cz128vq0535) (on [hgl-dev](https://hgl-dev.lib.harvard.edu/catalog/stanford-cz128vq0535))
      * Active tab styled like small black button; inactive tab styled like outline button 
      * MODS tab has a restyled alert box (trueno font) and code (Courier/monospace font)
      *  ISO tab has restyled text and links that match with mockup
         * I also noticed a bug were clicking on a jumplink in the table of contents ends up closing the modal...seems to be related to the turbolinks issues as i tested it in development_map branch where turbolinks is removed and is working as expected
   * [FGDC and HTML tabs](https://localhost:3001/catalog/cugir-007741) (on [hgl-dev](https://hgl-dev.lib.harvard.edu/catalog/cugir-007741))
      * FGDC tab will look like ISO tab
      * HTML tab is an iframe
   * [Not available message](https://localhost:3001/catalog/columbia-columbia-landinfo-global-aet) (on [hgl-dev](https://hgl-dev.lib.harvard.edu/catalog/columbia-columbia-landinfo-global-aet))
      * HTML tab is also empty in this example
   *  [Problem with request message](https://localhost:3001/catalog/02236876-9c21-42f6-9870-d2562da8e44f)
* More facets modal (examples on [local](https://localhost:3001/?f%5Bdc_subject_sm%5D%5B%5D=Boundaries&search_field=all_fields) and [hgl-dev](https://hgl-dev.lib.harvard.edu/?f%5Bdc_subject_sm%5D%5B%5D=Boundaries&search_field=all_fields)) - to view modal, click on "More link" under the subjects category
   * Selected facet in modal window has same styles as selected facet in the filter rail (black, bold)
   * Similarly, facet count is gray and bold
   * Previous/next buttons styled as links (no more thin colored border when hover over an enabled button)

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? no
- integration tests? no

# Interested parties
@phil-plencner-hl 
@enriquediaz
